src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private int firstSide;
+    private int secondSide;
+    private int heightIsoscelesTrapezoid;
+
+    public IsoscelesTrapezoid(String color, int firstSide, int secondSide,
+                              int heightIsoscelesTrapezoid) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        this.heightIsoscelesTrapezoid = heightIsoscelesTrapezoid;
```

Comment:
Update the constructor parameter name for the height based on the suggested change in line 7 🚧